tools/arriba_fusion_2.2.1.cwl

```diff
@@ -0,0 +1,58 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: arriba_fusion_2-2-1
+label: "Arriba Fusion Caller v2.2.1"
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/arriba:2.2.1'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    coresMin: 8
+    ramMin: ${ return inputs.memory * 1000 }
+
+baseCommand: [/arriba_v2.2.1/arriba]
+arguments:
+  - position: 0
+    shellQuote: false
+    valueFrom: >-
+      -o $(inputs.outFileNamePrefix).arriba_2.2.1.fusions.tsv
+      -O $(inputs.outFileNamePrefix).arriba_2.2.1.discarded_fusions.tsv
+
+inputs:
+  genome_aligned_bam: { type: File, doc: "STAR-aligned, coordinate sorted bam file",
+  secondaryFiles: [ { pattern: ".bai", required: false },  { pattern: "^.bai", required: false } ],
+  inputBinding: { prefix: '-x', position: 1 } }
+  memory: { type: 'int?', doc: "Mem intensive tool. Set in GB", default: 64 }
```

Comment:
Crazy that it uses so much memory on so few cores.

Reply:
Yup!